cocos/core/animation/playable.ts

```diff
@@ -23,6 +23,15 @@ export class Playable {
     get isPaused () {
         return this._isPaused;
     }
+
+    /**
+     * @en Is paused or stopped?
+     * @zh 是否已被暂停或停止。
+     */
+    get isMotionless () {
```

Comment:
isStill or isStatic seems simpler

Reply:
"Still" means not moving clam and quiet,
"Motionless" means not in physicial motion,
"Static" can be ambiguous with "static model".